.gitignore

```diff
@@ -1,5 +1,6 @@
 .idea
 scripts/htpasswd
+.DS_Store
```

Comment:
Not too important but I think it's a MacOS file, not sure if we need this file to be merged

Reply:
Yeah, it's just a MacOS metadata file for Finder.
I am pretty sure we don't want to add it to the repo, and it's annoying to see it in git changes, so that's why I added it to .gitignore :)

Reply:
True, never mind it was .gitignore ^^